webform_strawberryfield.routing.yml

```diff
@@ -20,6 +20,14 @@ webform_strawberryfield.auth_autocomplete:
     _access: 'TRUE'
 webform_strawberryfield.nominatim:
   path: '/webform_strawberry/nominatim/{api_type}/{count}/{lang}'
+  defaults:
+    _controller: '\Drupal\webform_strawberryfield\Controller\NominatimController::handleRequest'
+    _format: json
```

Comment:
```suggestion
    _format: json
    api_type: 'search'
    count: 1
```

Reply:
What do you think, could we default also lang? to the default language?

Reply:
@DiegoPino having 2 optional params of different types ('en' vs. 2) won't work well.
e.g. `/webform_strawberry/nominatim/{api_type}/{lang}/{count}`
route `webform_strawberry/nominatim/search/3` -- is 3 the lang or the count?

If you want a default I think we should make lang a query string parameter and remove it from the route. You are already accounting for no `lang` in the Controller by fetching the currentLanguage of the site and using. We can keep doing that and remove from the route. That way there is no confusion in the route.